modules/theme-tools/social-menu/icon-functions.php

```diff
@@ -130,6 +130,8 @@ function jetpack_social_menu_social_links_icons() {
 			'behance.net'     => 'behance',
 			'codepen.io'      => 'codepen',
 			'deviantart.com'  => 'deviantart',
+			'discord.gg'      => 'discord',
+			'discordapp.com'  => 'discord',
```

Comment:
This looks great and I will be to approve it but would you mind adding Discord to `modules/widgets/social-icons.php` so the Social Icons Widget can also benefit from this addition.

Reply:
Thanks @thomasguillot, I wasn't sure if that could only be done once Discord had been added to the social icons repo, but it doesn't seem that's the case. I've added it now. :)

![gsfsgffgsdgsfd](https://user-images.githubusercontent.com/43215253/52223808-5fe07480-289e-11e9-885f-18659fe95e12.png)